Moderation-Policy.md

```diff
@@ -0,0 +1,113 @@
+
+### Policy
+
+* All Posts are expected to respect the [Node.js Code of Conduct](https://GitHub.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md).
+* Only Collaborators with commit rights to a given repository may Moderate Posts within that repository's issue tracker.
+* Only a TSC member may Ban an individual from the Node.js GitHub Organization.
+* Any individual Banned from the Node.js GitHub Organization will be recommended for exclusion from any Node.js Foundation sponsored event or activity.
+
+#### Collaborator Posts
+
+* Collaborators must not Moderate any Post authored by another Collaborator without first giving the author at least 24 hours (from the time of the initial request) to modify or remove the Post on their own.
+* If the author of the Post disagrees that Moderation is required, the matter can be escalated to the TSC by creating a new issue (or using an existing one) within the nodejs/moderation repository with the label `tsc-agenda`. In such cases, the TSC will serve as the final arbiter.
+* While Moderation of a Collaborator's Post is in dispute, no Moderation action should be taken until a decision by the TSC is made.
+* In extreme circumstances involving either obvious gross violations of the [Node.js Code of Conduct](https://GitHub.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md) or possible compromise of a Collaborator's GitHub account, the TSC can be consulted to waive the 24 hour grace period and dispute process.
+* For Moderation issues involving a TSC member, the member in question is expected to recuse themselves from any decisions required to resolve the issue.
+* When Moderating any Post authored by another Collaborator, the moderating Collaborator must:
+ * Explain the justification for Moderating the post,
+ * Identify all changes made to the Post, and
+ * Identify the steps previously taken to resolve the issue.
+* Explanations of Moderation actions on Collaborator Posts must be provided in:
+ * A new post within the original thread, or
+ * A new issue within the private nodejs/moderation repository.
+* Any Collaborator who habitually authors Posts that must be Moderated will be subject to possible removal from the Node.js GitHub Organization. Such action can only be taken through normal TSC motion and vote.
+
+#### Non-Collaborator Posts
+
+* Posts authored by non-Collaborators are always subject to immediate Moderation by any Collaborator if the content is intentionally disruptive or in violation of the [Node.js Code of Conduct](https://GitHub.com/nodejs/node/blob/master/CODE_OF_CONDUCT.md).
+* When Moderating non-Collaborator Posts, the moderating Collaborator should:
+ * Explain the justification for Moderating the post, and
+ * Identify all changes made to the Post.
+* If an explanation of a Moderation action for a non-Collaborator Post is provided, it must be provided in:
```

Comment:
Maybe change `must` to `should`? The `If` implies that an explanation is not mandatory, so the set of methods for it is probably best express as a recommendation